Podcast/Util.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Util.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/7/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit 
+
+class Util {
+    
+    //format date interval between fromDate to toDate by greatest component between
+    // i.e. if fromDate: March 7, 1989  toDate: March 8, 1989 
+    //      formatDateDifferenceByLargestComponent(fromDate, toDate) -> "1 day ago"
+    //      if fromDate: March 9, 2015  toDate: March 8, 2017
+    //      formatDateDifferenceByLargestComponent(fromDate, toDate) -> "2 years ago"
+    static func formatDateDifferenceByLargestComponent(fromDate: Date, toDate: Date) -> String {
```

Comment:
Util is a very vague class, I'm worried we might just use it as a broom closet for random functions, better to put them as extensions, since this basically takes two dates and outputs a string